dim-web/src/lib.rs

```diff
@@ -1,19 +1,16 @@
-#![deny(warnings)]
-
 use std::future::IntoFuture;
 use std::net::SocketAddr;
 
 pub mod routes;
 pub mod tree;
 
 pub use axum;
-use axum::extract::{ConnectInfo, State};
+use axum::extract::{ConnectInfo, DefaultBodyLimit, State};
 use axum::response::Response;
-use axum::routing::{get, post, delete};
+use axum::routing::{get, patch, post, delete};
```

Comment:
In this codebase, we prefer to make each import a new line and avoid bulk imports.